.github/workflows/release.yml

```diff
@@ -6,16 +6,43 @@ on:
       version:
         description: 'The version that is going to be released. Should be in format 7.y.z'
         required: true
-        default: '7.y.z'
+        default: ''
+      forceRecreateTags:
+        description: If true, tags will be recreated. Use with caution
+        required: false
+        default: 'false'
 jobs:
   build:
     name: Create Release
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Check existing tags
+        run: |
+          RECREATE_TAGS=${{ github.event.inputs.forceRecreateTags }}
+          git ls-remote --exit-code origin refs/tags/${{ github.event.inputs.version}}
+          TAG_EXISTS=$?
+          if [[ $TAG_EXISTS -eq 0 ]]; then
+            if [[ $RECREATE_TAGS == "true" ]]; then
+              echo "[INFO] Recreating tags for ${{ github.event.inputs.version}} version"
+              git push origin :${{ github.event.inputs.version}}
+            else
+              echo "[ERROR] Cannot proceed with release - tag already exists"
```

Comment:
if the tag already exists... doesn't that mean that we have already released che operator for this $VERSION and therefore it's a GOOD thing that the docs are already done? 

Also should this bash be inside make-release.sh so we can run it locally w/o it needing to be run via an action? For this specific case it's probably not urgent, but I'm wondering if best practices here would be to minimize bash code embedded in yaml when we can simply call a bash script w/ params, so that it's easier to run that same script should ther ebe a GH outage.

Reply:
in future, perhaps we could move this to special reusable github action